### PR TITLE
Customize registration form to make email field unique

### PR DESCRIPTION
**Pull request by hyesun03**
* Make email field unique
* Add 'fullName' field
* Move the terms field to forms.py
* Remove functional test

### CODE REVIEW

File: kboard/accounts/models.py [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+    )
+    fullName = models.CharField(max_length=150)
+    status = models.CharField(max_length=20)
+
+    # Automatic timestamps
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
```
Comment (cjh5414) [inline kboard/accounts/models.py]: please use abstract model in core / models.py

File: kboard/kboard/urls.py [hunk 2]
```diff
@@ -15,10 +15,18 @@
...
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
     url(r'^summernote/', include('django_summernote.urls')),
+    url(r'^accounts/register/$',
+            RegistrationView.as_view(
+                form_class=RegistrationForm
+            ),
+            name='registration_register',
+            ),
```
Comment (guswnsxodlf) [inline kboard/kboard/urls.py]: Add this on accounts/urls.py

**Comment by guswnsxodlf** [comment]
LGTM

**Comment by cjh5414** [comment]
LGTM

**Pull request merged**